tests/integration/components/au-checkbox-group-test.js

```diff
@@ -0,0 +1,69 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'dummy/tests/helpers';
+import { click, render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | au-checkbox-group', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it can be given a value', async function (assert) {
+    this.groupValue = ['bar'];
+    await render(hbs`
+      <AuCheckboxGroup @value={{this.groupValue}} as |Group|>
+        <Group.Checkbox @name="foo" data-test-foo>Foo</Group.Checkbox>
```

Comment:
Should we also go for `@value` here instead of `@name`?

Reply:
> Should there perhaps also be an option for a group name? For the sake of HTML semantics? (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#handling_multiple_checkboxes as example)

I found the following inside the story for `AuCheckbox`: `name: { control: 'text', description: 'Groups checkboxes for server side' }`, this seems like something I copied from the old checkbox component but I sure seems logical, right?

Reply:
I've also added `@name` to the checkbox & checkbox-group component, but, for the latter, only as a custom value, not an automatic one (as we do for radio-group).